pkg/crc/constants/constants_darwin.go

```diff
@@ -12,7 +12,7 @@ const (
 )
 
 var (
-	TrayExecutablePath   = filepath.Join(filepath.Dir(filepath.Dir(version.GetMacosInstallPath())), "MacOS", "CodeReady Containers")
+	TrayExecutablePath   = filepath.Clean(filepath.Join(version.GetMacosInstallPath(), "..", "MacOS", "CodeReady Containers"))
```

Comment:
https://play.golang.org/p/IOyL1dQXi5A looks like this will change the path of installer side since we are using `/Applications/CodeReady Containers.app/Contents/Resources/` In the makefile and result for both implementation is different.

Reply:
> https://play.golang.org/p/IOyL1dQXi5A looks like this will change the path of installer side

The link you shared is missing one call to `filepath.Dir()`, `filepath.Dir()` is called twice: https://play.golang.org/p/Q3aDTmkM0uY